### ISSUE TITLE
add basic background soundtrack for ambiance

### ISSUE THREAD

**Issue by cvan** [mozilla/unity-webvr-export#123]
some nice ones:

- https://actuatorgames.itch.io/the-metis-one-original-soundtrack
- https://nicoleproducer.itch.io/spy-8-bit-16-bit
- https://itch.io/soundtracks/free

take your pick, @caseyyee!

**Comment by caseyyee** [comment]
good idea :)

**Comment by delapuente** [comment]
Yeah, great. I'm going to include some background sound instead of music. Background sound is less annoying (although your resources are great, @cvan!).
